plugins.html

```diff
@@ -340,10 +340,10 @@ <h1>Kanboard Plugins</h1>
                     <p>Greenwing is a kanboard beautify theme with a modern design.</p>
                     <p><em>By Confexion</em></p>
                 </dd>
-                <dt><a href="https://github.com/creecros/group_assign">Group assignment for tasks</a></dt>
+                <dt><a href="https://github.com/Jieiku/Group_assign">Group assignment for tasks</a></dt>
```

Comment:
@creecros Do you confirm that your plugin "group_assign" is now maintained by @Jieiku ?

Reply:
I did not formally ask them. However I seen a recent post by them that they no longer have the time to maintain their plugins.

I would also be fine with being a maintainer on their repository, which I asked about when I opened this pull request:

https://github.com/creecros/Group_assign/pull/65

That pull request was open for about 6 weeks with no response, I think they are just far too busy.

Hopefully you will get a response.

Reply:
Here is the post I was referring to: https://kanboard.discourse.group/t/creecros-plugins/2074

Reply:
I see I should have posted to that thread, maybe they would see me there. I will go try doing that now.

Reply:
If they do add me as a maintainer I will push to this once more using their github repo instead of mine.